src/client/components/componentWithSliders/savePressetModalBlock/modalSavePressetesWindow/modalSavePressetesWindow.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './modalSavePressetesWindow.css';
+import { cancelWindowIcon } from '../../../../assets/icons/icons';
+import SavePressetsToBDAndCancelButton from './savePressetsToBDAndCancelButton';
+
+const SavePressetesModalWindow = (props) => {
+  const { saveButtonClick, showHideModalBlock, refFocus } = props;
```

Comment:
You can destruct here also props in function parameters, please change this in all functional components

Reply:
destructed